test/cluster/test_read_repair.py

```diff
@@ -338,14 +259,18 @@ def check_rows(cql: Session, host: Host, expected_live_rows: row_set) -> None:
             actual_dead_rows -= actual_live_rows
 
             assert actual_live_rows == expected_live_rows
-            assert len(actual_live_rows) + len(actual_dead_rows) == total_rows
+            assert len(actual_live_rows) + len(actual_dead_rows) == total_rows - (len(all_rows) - len(actual_live_rows))
 
-        logger.info("Check rows with CL=ONE before read-repair")
+        # Need reconnecting after start/stop of nodes
+        await manager.driver_connect()
+        cql, hosts = await manager.get_ready_cql(nodes)
+
+        logger.info("Check rows with CL=LOCAL_ONE before read-repair")
```

Comment:
I don't see consistency_level passed or used anywhere in check_rows

Reply:
We shouldn't depend on LOCAL_ONE as the default.
If the test relies on it, it should pass it as a parameter (maybe initialized by default to LOCAL_ONE, but it would be visible in the interface), and use it for a SimpleStatement built in check_rows

Reply:
`check_rows()` uses `SELECT * FROM MUTATION_FRAGMENTS` which is `LOCAL_ONE` by definition. I will change the comment to not mention `LOCAL_ONE`.

Reply:
Comments fixed in v3.